CrossAppLoginUI/src/main/kotlin/com/infomaniak/core/crossloginui/views/CrossLoginListAccountsView.kt

```diff
@@ -0,0 +1,117 @@
+    defStyleAttr: Int = 0
+) : AbstractComposeView(context, attrs, defStyleAttr) {
+
+    private val accounts = mutableStateListOf<CrossLoginUiAccount>()
+
+    private var onAccountClickedListener: ((account: CrossLoginUiAccount) -> Unit)? = null
+    private var onAnotherAccountClickedListener: (() -> Unit)? = null
+    private var onCloseClicked: (() -> Unit)? = null
+
+    private var primaryColor by mutableStateOf<Color?>(null)
+    private var onPrimaryColor by mutableStateOf<Color?>(null) // TODO
+    private var titleColor: Color? = null
+    private var descriptionColor: Color? = null
+    private var avatarStrokeColor: Color? = null
+    private var buttonStrokeColor: Color? = null
+
+    init {
+        context.obtainStyledAttributes(attrs, R.styleable.CrossLoginListAccountsView, defStyleAttr, 0).apply {
+            primaryColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginPrimaryColor)
+            onPrimaryColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginOnPrimaryColor)
+            titleColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginTitleColor)
+            descriptionColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginDescriptionColor)
+            avatarStrokeColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginAvatarStrokeColor)
+            buttonStrokeColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginButtonStrokeColor)
+            recycle()
+        }
+    }
+
+    @Composable
+    override fun Content() {
+        MaterialThemeFromXml {
+
+            val colors = CrossLoginDefaults.colors(
+                primary = primaryColor,
+                onPrimary = onPrimaryColor,
+                title = titleColor,
+                description = descriptionColor,
+                avatarStroke = avatarStrokeColor,
+                buttonStroke = buttonStrokeColor,
+            )
+
+            CrossLoginListAccounts(
+                accounts = { accounts },
+                colors = colors,
+                onAccountClicked = { onAccountClickedListener?.invoke(it) },
+                onAnotherAccountClicked = { onAnotherAccountClickedListener?.invoke() },
+                onCloseClicked = { onCloseClicked?.invoke() },
+            )
+        }
+    }
+
+    fun setPrimaryColor(@ColorInt newPrimaryColor: Int) {
+        primaryColor = Color(newPrimaryColor)
+    }
+
+    fun setOnPrimaryColor(@ColorInt newOnPrimaryColor: Int) {
+        onPrimaryColor = Color(newOnPrimaryColor)
+    }
+
+    fun setAccounts(list: List<CrossLoginUiAccount>) {
+        accounts.apply {
+            clear()
+            addAll(list)
+        }
+    }
+
+    fun setOnAccountClickedListener(listener: (CrossLoginUiAccount) -> Unit) {
+        onAccountClickedListener = listener
+    }
+
+    fun setOnAnotherAccountClickedListener(listener: () -> Unit) {
+        onAnotherAccountClickedListener = listener
+    }
```

Comment:
I just realized that where this method is called you do a `.apply { forEach ...` when you could've done a `.onEach` instead, just to let you know

Reply:
You've placed a thumbs up but did not change the code. I'm not sure what this is supposed to mean. I'm getting mixed signals

Reply:
The code is in another PR.